p4-16/spec/P4-16-spec.mdk

```diff
@@ -3025,15 +3025,17 @@ The grammar production rule for general expressions is as follows:
 ~ Begin P4Grammar
 expression
     : INTEGER
+    | DOTS                           // DOTS is ...
+    | STRING_LITERAL
     | TRUE
     | FALSE
-    | STRING_LITERAL
-    | nonTypeName
-    | dotPrefix nonTypeName
+    | prefixedNonTypeName
```

Comment:
The definition of `prefixedNonTypeName` is `(nonTypeName | dotPrefix nonTypeName)`, so this and the similar change in grammar.mdk are not a change in the syntax.  This changed version is what p4c uses.